core/src/templates/components/global-footer/global-footer.twig

```diff
@@ -11,14 +11,19 @@
 {% if template_path_logo is empty -%}
   {%- set template_path_logo = "@decanter/components/logo/logo.twig" -%}
 {%- endif %}
+{% if external_link_text is empty -%}
+  {%- set external_link_text -%}
+    <span class="su-global-footer__link-a11y">(link is external)</span>
+  {% endset %}
+{%- endif %}
 
 <div class="su-global-footer {{ modifier_class }}">
-  <div class="su-global-footer__container">
+  <div class="su-global-footer__container" title="Common Stanford resources"|t>
```

Comment:
Ignore my previous comments about string translation. That is only for Drupal and is not supported here. My bad.
```suggestion
  <div class="su-global-footer__container" title="Common Stanford resources">
```